databroker/src/grpc/server.rs

```diff
@@ -165,23 +174,35 @@ where
         info!("Authorization is not enabled.");
     }
 
-    builder
-        .add_service(
+    let kuksa_val_v1 = {
+        if apis.contains(&Api::KuksaValV1) {
+            Some(kuksa::val::v1::val_server::ValServer::with_interceptor(
```

Comment:
val_server can maybe cause confusion with old kuksa_val_server in c++?

Reply:
This was not changed as part of this commit. It is referencing generated code, which having a protobuf service definition like this:

```proto
service VAL {
  ...
}
```

will generate code `<camel case service name>_server::<camel case service name>Server`, i.e. `val_server::ValServer`.